src/pages/DashboardPage/DashboardPage.css

```diff
@@ -9,4 +9,14 @@
     display: flex;
     flex-wrap: wrap;
     justify-content: center;
+}
+
+.dashboard-no-entries-container {
+    display: flex;
+    flex-direction: column;
+    max-width: min(80ch, 100%);
```

Comment:
Useful for displaying lots of text, makes sure the width isn't too large so that the reader doesn't lose interest